.gitignore

```diff
@@ -80,7 +80,14 @@ target/
 
 ### Django ###
 *.log
+<<<<<<< HEAD
```

Comment:
Please, fix the merge conflict: remove HEAD (line 83) and >>>>>>> 1c8755fca3e1bfa08a0541c3375bd3ecc015745c (line 90)

Reply:
sorry, fixed right now